test/e2e/common_suite_test.go

```diff
@@ -236,6 +254,39 @@ func newTestCase(t *testing.T, testName string, assert CloudAssert, assessMessag
 
 	return testCase
 }
+func ComparePodLogString(ctx context.Context, t *testing.T, client klient.Client, Pod v1.Pod, ExpectedPodlogString string) (bool, string, error) {
```

Comment:
Is this function intended to be exported? otherwise I think it should start with a lower case too?